docs/concepts/bulk-update.md

```diff
@@ -0,0 +1,169 @@
+| Patient Breed Description | (0010,2292) | The breed of the patient.See Section C.7.1.1.1.1.  |
+| Breed Registration Number | (0010,2295) | Identification number of a veterinary patient within the registry.  |
+
+> Maximum of 50 studies can be updated at once.
+
+> Only one update operation can be performed at a time.
+
+### Get Operation
+Get a long-running operation.
+
+```http
+GET .../operations/{operationId}
+```
+
+#### URI Parameters
+
+| Name        | In   | Required | Type   | Description      |
+| ----------- | ---- | -------- | ------ | ---------------- |
+| operationId | path | True     | string | The operation id |
+
+#### Responses
+
+```json
+{
+    "operationId": "1323c079a1b64efcb8943ef7707b5438",
+    "type": "update",
+    "createdTime": "2023-05-08T05:01:30.1441374Z",
+    "lastUpdatedTime": "2023-05-08T05:01:42.9067335Z",
+    "status": "completed",
+    "percentComplete": 100,
+    "results": {
+        "studyUpdated": 1,
+        "instanceUpdated": 16,
+        // Errors will go here
+    }
+}
+```
+
+| Name            | Type                    | Description                                  |
+| --------------- | ----------------------- | -------------------------------------------- |
+| 200 (OK)        | [Operation](#operation) | The completed operation for the specified ID |
+| 202 (Accepted)  | [Operation](#operation) | The running operation for the specified ID   |
+| 404 (Not Found) |                         | The operation is not found                   |
+
+## Retrieve (WADO-RS)
+
+The Retrieve Transaction feature provides the ability to retrieve stored studies, series, and instances, including both the original and latest versions.
+
+| Method | Path                                                                    | Description |
+| :----- | :---------------------------------------------------------------------- | :---------- |
+| GET    | ../studies/{study}                                                      | Retrieves all instances within a study. |
+| GET    | ../studies/{study}/metadata                                             | Retrieves the metadata for all instances within a study. |
+| GET    | ../studies/{study}/series/{series}                                      | Retrieves all instances within a series. |
+| GET    | ../studies/{study}/series/{series}/metadata                             | Retrieves the metadata for all instances within a series. |
+| GET    | ../studies/{study}/series/{series}/instances/{instance}                 | Retrieves a single instance. |
+| GET    | ../studies/{study}/series/{series}/instances/{instance}/metadata        | Retrieves the metadata for a single instance. |
+
+In order to retrieve original version, `msdicom-request-original` header should be set to `true`.
+
+```
+- `msdicom-request-original=true
+```
+
+## Delete
+
+This transaction will delete both original and latest version of instances.
```

Comment:
I get why this is included here, but it feels a bit out of place.  Maybe call out the limitation (which I assume exists) that there is no way to delete the latest version or revert back to the original.  

Also, the conformance statement should probably be updated to include this note in the Delete section.